config/core/test-configs.yaml

```diff
@@ -2757,6 +2762,16 @@ test_configs:
     test_plans:
       - baseline
 
+  - device_type: rk3399-roc-pc
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - igt-gpu-panfrost
```

Comment:
@montjoie This test plan is not enabled in `lab-clabbe`:
```yaml
    filters:
      - passlist:
          plan:
            - baseline
            - sleep
```
The board is not online in `lab-broonie` yet so we don't have any results for Panfrost.

Reply:
OK thanks, and `lab-baylibre` doesn't have `igt` or `kselftest` enabled so I think we should just drop them for now so we can merge this PR without any dead code / config.